test/hotspot/jtreg/runtime/CompressedOops/CompressedCPUSpecificClassSpaceReservation.java

```diff
@@ -28,6 +28,7 @@
  * @requires vm.bits == 64 & !vm.graal.enabled & vm.debug == true
  * @requires vm.flagless
  * @requires vm.cds
+ * @requires vm.cds.default.archive.available
```

Comment:
Does the notion of "default.archive" change depending on whether the VM is executing in COOPs/NoCOOPS or COH/NoCOH modes?

Reply:
That's the coding 

https://github.com/openjdk/jdk/blob/320235ccb88b4b554cd5756b7e6a34ce97aabd53/test/jtreg-ext/requires/VMProps.java#L438

So it ALWAYS looks at classes,jsa .

Reply:
Okay ... so what configuration does classes.jsa represent? I'm struggling to understand exactly which archive these tests need, or whether they should be able to work with any of the 4 variants. ??

Reply:
Probably the requires vm.cds.default.archive.available  is from times with less jsa files then we usually have today . We could of course introduce 4 requires for all the different  jsa's we potentially have these days but I am not sure if it is really needed,